packages/app/components/Queue/QuestionForm.tsx

```diff
@@ -66,13 +67,38 @@ export default function QuestionForm({
   // on question type change, update the question type state
   const onCategoryChange = (e: RadioChangeEvent) => {
     setQuestionTypeInput(e.target.value);
+
+    const questionFromStorage = JSON.parse(
+      window.localStorage.getItem("draftQuestion")
```

Comment:
why do you use `localStorage` instead of state?

Reply:
wait a second. I think it'd make most sense to get it from the server. Currently, when you draft a question, we create a question server side and set its status to `Drafting` so we should be able to just figure out whether you have a draft in progress by looking through the question list and filtering for questions created by current user with status = drafting.

Reply:
No local state should be used at all, because we're just synching with the server.

Reply:
But I don't think it makes any sense to save a question to the serve that many times. Like sure for the category change it might make sense. But when they're midtyping their question and then the internet cuts out or their browser just shuts down? @liustanley, @ofloody, and I discussed this before starting that it makes sense for the server to know that there is a draft question, but for us not to back and forth save the question repeatedly.

Reply:
i agree with aislin, pinging the server on every value change to the input box seems excessive